helm/csi-powermax/templates/controller.yaml

```diff
@@ -402,6 +402,20 @@ spec:
               value: "{{ .Values.controller.healthMonitor.enabled }}"
             {{- end }}
             {{- end }}
+            - name: X_CSI_VSPHERE_ENABLED
+              value: "{{ .Values.vSphere.enabled }}"
+            {{- if eq .Values.vSphere.enabled true }}
+            - name: X_CSI_VSPHERE_PORTGROUP
+              value: {{ required "Must provide portgroup for vsphere" .Values.vSphere.fcPortGroup }}
+            - name: X_CSI_VSPHERE_HOSTGROUP
+              value: {{ required "Must provide portgroup for vsphere" .Values.vSphere.fcHostGroup }}
```

Comment:
required message contains same value for all attributes

Reply:
updated message .